test/check-bootstrap.bash

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function line() {
+    echo ================================================================================
+}
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do
+  DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE"
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
+
+if [ $# != 3 ]; then
+    echo "Usage: check-bootstrap.bash <target> <Aeneas before> <Aeneas after>"
```

Comment:
what about accepting a glob of filenames (e.g. execute/\*.v3) as the rest of the arguments, and defaulting to ./\*.v3, like the test.bash scripts do?

Reply:
Done.